social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOC_People_Activity_CommentTestIT.java

```diff
@@ -200,6 +200,7 @@ public void test02_CommentOnYourFriendsActivity() {
     $(byId("cke_CommentTextarea" + id)).waitUntil(Condition.appears, Configuration.timeout).click();
     executeJavaScript("CKEDITOR.instances.CommentTextarea" + id + ".insertText(\"" + comment + "\")", "");
     // click on the button comment
+    Utils.pause(2000);
```

Comment:
Isn't one of the purposes of selenide to avoid adding pauses explicitly ? (that's a true question, I did not dig a lot in selenide for the moment :) )